app/views/checkout/payment.scala.html

```diff
@@ -53,7 +51,7 @@
         <form class="form js-checkout-form" action="@routes.Checkout.renderCheckout(CountryGroup.UK, None).toString" method="POST" novalidate>
             @helper.CSRF.formField
 
-            @fragments.checkout.checkoutHeader("Checkout")
+            @fragments.checkout.checkoutHeader("Digital Pack", promoCode.exists(_.get == "DGA85"))
```

Comment:
This is a temporary measure, for one particular promo code, to let us see if it improves conversion. If it's successful I'll remove the test and @if around the content in the template file.